### PR TITLE
Render unordered lists

### PR DESCRIPTION
**Pull request by webb04**
## Why are you doing this?
UL elements weren't being rendered.

## Changes
- Moved more bullet styles over to `renderer.ts`

## Screenshots

| Before | After |
| --- | --- |
|  |  |

### CODE REVIEW

File: src/renderer.ts [hunk 1]
```diff
@@ -50,6 +50,32 @@ const Anchor = (props: { href: string; text: string; pillar: Pillar }): ReactEle
...
+    &::before {
+        display: inline-block;
+        content: '';
+        border-radius: 0.5rem;
+        height: 1rem;
+        width: 1rem;
+        margin-right: 1rem;
+        background-color: ${palette.neutral[86]};
+        margin-left: -2rem;
+    }
```
Comment (JamieB-gu) [inline src/renderer.ts]: I think if you use `list-style-position: inside` in `listStyles` then you will be able to drop the `padding-left: 2rem` and `margin-left: -2rem` in `listItemStyles`.

File: src/renderer.ts [hunk 2]
```diff
@@ -50,6 +50,32 @@ const Anchor = (props: { href: string; text: string; pillar: Pillar }): ReactEle
...
+
+    > p:first-of-type {
+        display: inline;
+    }
```
Comment (JamieB-gu) [inline src/renderer.ts]: Out of curiosity, what does this do?

Comment (webb04) [reply src/renderer.ts]: `p` tags will be block and clear to the next line

File: src/renderer.ts (hunk 1 above)
Comment (webb04) [reply src/renderer.ts]: Ah that looked like an improvement but it's not working with out custom bullet points made from a pseudo ::Before

**Pull request merged**